cmd/boots/tftp.go

```diff
@@ -68,21 +68,21 @@ func (t tftpHandler) ReadFile(c tftp.Conn, filename string) (tftp.ReadCloser, er
 		l.Info("client requested filename '", filename, "' with a traceparent attached and has been shortened to '", shortfile, "'")
 		filename = shortfile
 	}
+
 	tracer := otel.Tracer("TFTP")
-	ctx, span := tracer.Start(ctx, "TFTP get",
+	// save the context in a different var so it can be used to create links later
+	rctx, span := tracer.Start(ctx, "TFTP get",
```

Comment:
I didn't notice this last time, but it would be nice to make this string match the tftp protocol's wording I think. Could you change `TFTP get` to `TFTP Read` instead? Not a big deal if you'd rather not we can address separately though.

Reply:
> 

will do, I was thinking about refactoring all these names now that we know more about how otel fits into the code

for now I'll update the bits in this PR and follow up later with another that refactors the rest into some normalcy, probably along the lines of the otel semantic conventions https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/rpc.md